src/path.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/**
+ * @fileoverview Path manipulation functions.
+ * These are the functions exposed by nodejs in the 'path' module.
```

Comment:
AlexE was talking about tsickle supporting multiple versions of the TypeScript compiler simultaneously because that's how Angular wants to use it. This might make this harder to achieve, but then again I don't think it is a good idea to begin with :/